Source/SpatialGDK/Public/Utils/SchemaOption.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
+
+#include "Platform.h"
```

Comment:
What is it in this file that requires `Platform.h`? You also need to add `UniquePtr.h`

Reply:
Was using it for `UniquePtr` but will just include the header instead.